src/bindings/python/local.cpp

```diff
@@ -36,111 +42,118 @@
-       return SWIGTYPE_p_SBMLLevelVersionConverter;
-     else if (conName == "SBML Initial Assignment Converter")
-       return SWIGTYPE_p_SBMLInitialAssignmentConverter;
-     else if (conName == "SBML Infer Units Converter")
-       return SWIGTYPE_p_SBMLInferUnitsConverter;
-     else if (conName == "SBML Id Converter")
-       return SWIGTYPE_p_SBMLIdConverter;
-     else if (conName == "SBML Function Definition Converter")
-       return SWIGTYPE_p_SBMLFunctionDefinitionConverter;
+     if (conName == "LX Units Converter")
+       return SWIGTYPE_p_LXUnitsConverter;
+     else if (conName == "LX Strip Package Converter")
+       return SWIGTYPE_p_LXStripPackageConverter;
+     else if (conName == "LX Rule Converter")
+       return SWIGTYPE_p_LXRuleConverter;
+     else if (conName == "LX Reaction Converter")
+       return SWIGTYPE_p_LXReactionConverter;
+     else if (conName == "LX Local Parameter Converter")
+       return SWIGTYPE_p_LXLocalParameterConverter;
+     else if (conName == "LX Level Version Converter")
+       return SWIGTYPE_p_LXLevelVersionConverter;
+     else if (conName == "LX Initial Assignment Converter")
+       return SWIGTYPE_p_LXInitialAssignmentConverter;
+     else if (conName == "LX Infer Units Converter")
+       return SWIGTYPE_p_LXInferUnitsConverter;
+     else if (conName == "LX Id Converter")
+       return SWIGTYPE_p_LXIdConverter;
+     else if (conName == "LX Function Definition Converter")
+       return SWIGTYPE_p_LXFunctionDefinitionConverter;
 
 #include "local-downcast-converters.cpp"	
 	   
-	return SWIGTYPE_p_SBMLConverter;
+	return SWIGTYPE_p_LXConverter;
 }
-
+*/
 /**
- * @return the most specific Swig type for the given SBMLNamespaces object.
+ * @return the most specific Swig type for the given LXNamespaces object.
  */
+ /*
 struct swig_type_info*
-GetDowncastSwigType (SBMLNamespaces* se)
+GetDowncastSwigType (LXNamespaces* se)
 {
-	if (se == 0) return SWIGTYPE_p_SBMLNamespaces;
+	if (se == 0) return SWIGTYPE_p_LXNamespaces;
 	
 	const std::string pkgName = se->getPackageName();
 
 #include "local-downcast-namespaces.cpp"
 
-	return SWIGTYPE_p_SBMLNamespaces;
+	return SWIGTYPE_p_LXNamespaces;
 }
-
+*/
 /**
  * @return the most specific Swig type for the given SBasePlugin object.
  */
+ /*
 struct swig_type_info*
 GetDowncastSwigType (SBasePlugin* sbp)
 {
   if (sbp == 0) return SWIGTYPE_p_SBasePlugin;
 
   const std::string pkgName = sbp->getPackageName();
-  SBase* sb = sbp->getParentSBMLObject();
+  SBase* sb = sbp->getParentLXObject();
   if (!sb) return SWIGTYPE_p_SBasePlugin;
 	
 #include "local-downcast-plugins.cpp"
 
-  if (sb->getTypeCode() == SBML_DOCUMENT)
-    return SWIGTYPE_p_SBMLDocumentPlugin;
+  if (sb->getTypeCode() == LX_DOCUMENT)
+    return SWIGTYPE_p_LXDocumentPlugin;
 
   return SWIGTYPE_p_SBasePlugin;
 }
+*/
 
-
-struct swig_type_info*
-GetDowncastSwigTypeForPackage (SBase* sb, const std::string &pkgName);
+//struct swig_type_info*
```

Comment:
Why are these commented?

Reply:
This file was originally copied from the `libsbml` repository. I went through commenting-out functions etc which were needed for linSBML, but not needed for libLX. I suppose I should have deleted them, rather than commenting them out, but I was being cautious.